performance/pom.xml

```diff
@@ -202,7 +202,7 @@
             </activation>
             <properties>
                 <beanvalidation-impl.name>Hibernate Validator</beanvalidation-impl.name>
-                <beanvalidation-impl.version>6.0.7.Final</beanvalidation-impl.version>
+                <beanvalidation-impl.version>6.0.8.Final</beanvalidation-impl.version>
```

Comment:
wanted to ask if anybody knows of a nice way to do these kind of changes "automatically". I thought of something like:
```xml
<beanvalidation-impl.version>[6.0.0,${project.version})</beanvalidation-impl.version>
```
I tried it and it gives the result:
```
[INFO] --- maven-shade-plugin:3.1.0:shade (default) @ hibernate-validator-performance ---
.....
[INFO] Including javax.validation:validation-api:jar:2.0.1.Final in the shaded jar.
[INFO] Including org.hibernate.validator:hibernate-validator:jar:6.0.8.Final in the shaded jar.
....
```
but I guess it could be unclear. 
Also I've noticed the `previous.stable` property in main pom.xml :
```xml
<previous.stable>6.0.4.Final</previous.stable>
```
should we update this one? and maybe we should use it in performance tests pom as well ?

Reply:
No good idea here. The fact is that they might be different as `previous.stable` could refer to the last version of the previous major.minor release whereas we might want to test the performance against the latest micro.